db/Models/news.models.js

```diff
@@ -20,4 +20,24 @@ const selectApi = () => {
   return Promise.resolve(endpoints);
 };
 
-module.exports = { selectTopics, selectApi };
+const selectArticle = (article_id) => {
+  let queryString = `SELECT * FROM articles`;
+  let queryValues = [];
+  //   console.log(article_id);
+  if (Number(article_id)) {
+    queryString += ` WHERE article_id = $1`;
+    queryValues.push(Number(article_id));
+  } else {
+    return Promise.reject({ status: 400, msg: "Bad request" });
+  }
+  //   console.log(queryString);
+  return db.query(queryString, queryValues).then((data) => {
+    if (data.rows.length === 0) {
+      return Promise.reject({ status: 400, msg: "Bad request" });
```

Comment:
This wouldn't be a 400 if there are no rows in the database. This would be a 404, a valid search that had no results.